scripts/run-integration-tests.travis.sh

```diff
@@ -2,15 +2,15 @@
 set -ev
 docker build -t one-app:at-test .
 if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then
-  npm run build:sample-modules -- --archive-built-artifacts --bundle-statics-origin=https://one-app-statics.surge.sh
-  echo 'https://one-app-sample.herokuapp.com' >> sample-module-bundles/CORS && \
-  npx surge teardown one-app-statics.surge.sh && \
-  npx surge sample-module-bundles one-app-statics.surge.sh && \
+  npm run build:sample-modules -- --archive-built-artifacts --bundle-statics-origin=$SURGE_DOMAIN
```

Comment:
Can we make these more _generic_? If in the future we go away from surge/heroku we won't have to rename these.

Reply:
Unfortunately, the script is very specific on how surge and Heroku deployments work, if we need to use a different deployment system, it will likely require different variables / not require them at all